packages/patternfly-4/react-core/src/components/Tabs/Tab.d.ts

```diff
@@ -0,0 +1,11 @@
+import { SFC, HTMLProps } from 'react';
+
+export interface TabProps extends HTMLProps<HTMLDivElement> {
+  children: any;
```

Comment:
We don't need children as that is included in html props.

Reply:
I removed this.